### PR TITLE
Add utils to the wp-data script dependencies.

### LINKED ISSUE #10102: Gutenberg 3.9: Uncaught TypeError: Cannot read property 'uid' of undefined
**Describe the bug**

**To Reproduce**
1. Update plugin to Gutenberg 3.9
2. Refresh browser (Vanilla TwentySeventeen). No other plugins
3. See Error: Uncaught TypeError: Cannot read property 'uid' of undefined, line 69 of client-assets.php
4. Offending line: var userId = window.userSettings.uid;

**Expected behavior**
NOT to see that.

**Desktop (please complete the following information):**
 - OS: Windows 7
 - Browser Chrome

**Smartphone (please complete the following information):**
 - Version 69.0.3497.100 (Official Build) (64-bit)

### PR DESCRIPTION
**Pull request by kopepasah**
The `wp-data` inline script relies on `window.userSettings.uid`, a property set in the `utils` script.

Fixes #10102

## Description
Added `utils` as a script dependencies for `wp-data`.

## How has this been tested?
Tested locally using WordPress `5.0-alpha-43661` and Gutenberg `master` with HEAD at `7ec834b77686c6b291ae1ee18fe5860714b58236`.

## Types of changes
This is a bug fix for #10102.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [ ] My code has proper inline documentation.

### CODE REVIEW

File: lib/client-assets.php [hunk 1]
```diff
@@ -246,6 +246,7 @@ function gutenberg_register_scripts_and_styles() {
 		'wp-data',
 		gutenberg_url( 'build/data/index.js' ),
 		array(
+			'utils',
```
Comment (youknowriad) [inline lib/client-assets.php]: Maybe we can use `get_current_user_id` in the inline script to avoid loading another script just for a variable?

Comment (kopepasah) [reply lib/client-assets.php]: @youknowriad the `utils` script is a dependency for other scripts (`/wp-includes/js/media-views.js`, `/wp-admin/js/common.js`, `/wp-admin/js/editor.js`) and will load anyway, it's just a matter of timing. 

However, if the intention is to move away from older libraries, then using just `get_current_user_id()` in the inline script makes sense.

Comment (youknowriad) [reply lib/client-assets.php]: Gotcha, I think I have a small preference towards avoiding the dependency. Maybe at some point, these dependencies won't be needed anymore.

Comment (kopepasah) [reply lib/client-assets.php]: Sounds good to me. I pushed up the change.

**Review by youknowriad — APPROVED** [review]
LGTM 👍 Let's merge when the tests pass.

**Pull request merged**